backend/app2/api/routes/users.py

```
@@ -0,0 +1,122 @@
+    Change user's password
+    """
+    logger.info(f"Changing password for user ID: {current_user['id']}")
+    try:
+        success = await user_service.change_password(
+            user_id=current_user["id"],
+            current_password=password_data.current_password,
+            new_password=password_data.new_password
+        )
+        logger.info(f"Password changed for user ID: {current_user['id']}")
+        return {"message": "Password changed successfully"}
+    except UnauthorizedException:
```

Ruff (0.8.2)

Undefined name `UnauthorizedException`

(F821)